server/schedule/operator/step.go

```diff
@@ -463,6 +536,15 @@ func (dv DemoteVoter) Timeout(start time.Time, regionSize int64) bool {
 	return time.Since(start) > fastStepWaitDuration(regionSize)
 }
 
+// GetCmd returns the schedule command for heartbeat response.
+func (dv DemoteVoter) GetCmd(region *core.RegionInfo, useConfChangeV2 bool) *pdpb.RegionHeartbeatResponse {
```

Comment:
`DemoteVoter` is not an `OpStep`. We don't need to implement this function.

Reply:
To make it consistent with others, we can simply call `GetCmd()` of it in `ChangePeerV2Enter`